sdk/tests/sdk/whirlpools/position-impl#collectFees.test.ts

```diff
@@ -178,13 +179,13 @@ describe("PositionImpl#collectFees()", () => {
 
       assert.notEqual(positionDataAfter, null);
 
-      const accountAPubkey = await deriveATA(otherWallet.publicKey, poolInitInfo.tokenMintA);
-      const accountA = await testCtx.whirlpoolCtx.fetcher.getTokenInfo(accountAPubkey, true);
-      assert.ok(accountA && accountA.amount.eq(quote.feeOwedA));
+      const accountAPubkey = getAssociatedTokenAddressSync(poolInitInfo.tokenMintA, otherWallet.publicKey);
+      const accountA: Account = await testCtx.whirlpoolCtx.fetcher.getTokenInfo(accountAPubkey, true);
```

Comment:
super nit: is ``: Account``  required?  (accountBPubkey is defined explicit data type)